src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -144,6 +144,8 @@
             + "The value of '${fieldName}' field should be no longer than ${maxLength} characters.";
     public static final String INVALID_NAME_ERROR_MESSAGE =
             ERROR_INFO + " " + HINT_FOR_CORRECT_FORMAT_FOR_INVALID_NAME;
+    public static final String TEAM_NAME_IS_VALID_EMAIL_ERROR_MESSAGE =
+            "The field " + TEAM_NAME_FIELD_NAME + " is not acceptable to TEAMMATES as it appears to be a valid email";
```

Comment:
"As it appears to be a valid email" does not tell the user immediately why it cannot be a team name.

Reply:
How about "is not acceptable to TEAMMATES as team name can't be a valid email"?